docs/content/contributing/_index.en.md

```diff
@@ -26,33 +26,49 @@ directory (if you want to build the image with the docker inside
 minikube, you'll need to set the proper environment variables with
 `eval $(minikube docker-env)`):
 
-```
-  # Get dependencies
-  $ glide install
+Install dependencies with glide:
 
-  # Build fission server and an image
-  $ pushd fission-bundle
-  $ ./build.sh
-```
+{{< highlight bash >}}
+$ glide install -v
+{{< /highlight >}}
+
+Build fission server:
+
+{{< highlight bash >}}
+$ pushd $GOPATH/src/github.com/fission/fission/fission-bundle
+$ ./build.sh
+{{< /highlight >}}
 
 You now need to build the docker image for fission. You can use
 `push.sh` and push it to a docker hub account. But it's easiest to use
 minikube and its built-in docker daemon:
 
-```
-  $ eval $(minikube docker-env)
-  $ docker build -t minikube/fission-bundle .
-```
+{{< highlight bash >}}
+$ eval $(minikube docker-env)
+$ docker build -t minikube/fission-bundle .
+{{< /highlight >}}
+
+Next, pull in the dependencies for the Helm chart:
+
+{{< highlight bash >}}
+$ helm dep update $GOPATH/src/github.com/fisson/charts/fission-all
+{{< /highlight >}}
 
 Next, install fission with this image on your kubernetes cluster using the helm chart:
 
-```
-  $ helm install --set "image=minikube/fission-bundle,pullPolicy=IfNotPresent,analytics=false" charts/fission-all
-```
+{{< highlight bash >}}
+$ helm install --set "image=minikube/fission-bundle,pullPolicy=IfNotPresent,analytics=false" charts/fission-all
+{{< /highlight >}}
 
 And if you're changing the CLI too, you can build it with:
 
-```
-  # Build Fission CLI
-  $ cd fission && go install
-```
+{{< highlight bash >}}
+$ cd $GOPATH/src/github.com/fission/fission/fission
```

Comment:
this results in

```
cd $GOPATH/src/github.com/fission/fission/fission
-bash: cd: /home/sdake/go/src/github.com/fission/fission/fission: No such file or directory
```

Reply:
It works for me. Would you mind to check the your repo directory location?
The root directory should be placed at `$GOPATH/src/github.com/fission/fission`

Reply:
ya that should work - depending on where the repo is checked out.

Typically I set an env variable such as `$ISTIO` to `$GOPATH/src/istio` leaving `$GOPATH/src/github.com` for vendored dependencies.

In the case of fission, I checked out fission to `$GOPATH/src/fission`.

I'll add some git clone documentation in this PR later this afternoon.

Cheers
-steve